src/nft/components/details/detailsV2/InfoChips.tsx

```diff
@@ -0,0 +1,126 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { Unicon } from 'components/Unicon'
+import useENSAvatar from 'hooks/useENSAvatar'
+import useENSName from 'hooks/useENSName'
+import { useScreenSize } from 'hooks/useScreenSize'
+import { GenieAsset } from 'nft/types'
+import { ReactNode, useState } from 'react'
+import { ChevronDown, ChevronUp, DollarSign } from 'react-feather'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+import { isAddress, shortenAddress } from 'utils'
+
+const StyledBubble = styled(Row)`
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  padding: 10px 12px 10px 8px;
+  border-radius: 20px;
+  max-width: 144px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    max-width: 169px;
+  }
+`
+
+const StyledLabelMedium = styled.div`
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 20px;
+  color: ${({ theme }) => theme.textPrimary};
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+`
+
+const StyledIcon = styled(Row)`
```

Comment:
Does this need to be a row with justify and align if it only contains 1 item?

Reply:
it aligns the svg in the center because the size of the svg is smaller than `StyledIcon`